src/qml/FeatureForm.qml

```diff
@@ -32,6 +32,7 @@ Page {
   property int embeddedLevel: 0
   //setupOnly means data would be neither saved nor cleared (feature creation is handled elsewhere like e.g. in the tracking)
   property bool setupOnly: false
+  property bool ignoreChanges: false
```

Comment:
This is an internal (private implementation detail)? Can we put this into the QtObject with the id master instead?
And ideally add a small comment for its purpose